sphinx/io.py

```diff
@@ -39,6 +43,9 @@
     from sphinx.environment import BuildEnvironment  # NOQA
 
 
+logger = logging.getLogger(__name__)
```

Comment:
Finally, this is not used. So I will remove this.

Reply:
Ah, this comment is wrong. It is used at `warn_and_replace`. So I'll keep this as is.